raiden/tests/scenarios/Scenario-Mediated-Transfer-in-Hexagon.yaml

```diff
@@ -0,0 +1,190 @@
+            - assert: {from: 4, to: 5, balance: 5, state: "opened"}
+            - assert: {from: 5, to: 6, balance: 5, state: "opened"}
+            - assert: {from: 6, to: 1, balance: 5, state: "opened"}
+
+            - assert: {from: 2, to: 1, balance: 5, state: "opened"}
+            - assert: {from: 3, to: 2, balance: 5, state: "opened"}
+            - assert: {from: 4, to: 3, balance: 5, state: "opened"}
+            - assert: {from: 5, to: 4, balance: 5, state: "opened"}
+            - assert: {from: 6, to: 5, balance: 5, state: "opened"}
+            - assert: {from: 1, to: 6, balance: 5, state: "opened"}
+
+            - assert: {from: 0, to: 1, balance: 5, state: "opened"}
+            - assert: {from: 0, to: 2, balance: 5, state: "opened"}
+            - assert: {from: 0, to: 3, balance: 5, state: "opened"}
+            - assert: {from: 0, to: 4, balance: 5, state: "opened"}
+            - assert: {from: 0, to: 5, balance: 5, state: "opened"}
+            - assert: {from: 0, to: 6, balance: 5, state: "opened"}
+
+            - assert: {from: 1, to: 0, balance: 5, state: "opened"}
+            - assert: {from: 2, to: 0, balance: 5, state: "opened"}
+            - assert: {from: 3, to: 0, balance: 5, state: "opened"}
+            - assert: {from: 4, to: 0, balance: 5, state: "opened"}
+            - assert: {from: 5, to: 0, balance: 5, state: "opened"}
+            - assert: {from: 6, to: 0, balance: 5, state: "opened"}
+
+      - parallel:
+          name: "Test 1: Node 1 tries to send 1 token to Node 4 but all possible paths crash"
+          tasks:
+            - serial:
+                name: "transfer"
+                tasks:
+                  - transfer: {from: 1, to: 4, amount: 1, expected_http_status: "(200|409)"}
+                  - wait: 30
+                  - assert_all: {from: 4, count: 3, balances: [6, 5, 5]}
+            - serial:
+                name: "Node 0 crashes"
+                tasks:
+                  - stop_node: 0
+                  - wait: 5
+                  - start_node: 0
+            - serial:
+                name: "Node 3 crashes"
+                tasks:
+                  - stop_node: 3
+                  - wait: 5
+                  - start_node: 3
+            - serial:
+                name: "Node 5 crashes"
+                tasks:
+                  - stop_node: 5
+                  - wait: 5
+                  - start_node: 5
+
+      - serial:
+          name: "Test 2: Node 1 transfers 1 token to Node 5, whereas Node 0 and Node 6 are shut down (there is only the long path)"
+          tasks:
+            - stop_node: 0
+            - stop_node: 6 #this is to block alternative paths
+            - parallel:
+                tasks:
+                  - serial:
+                      tasks:
+                        - transfer: {from: 1, to: 5, amount: 1, expected_http_status: "(200|409)"}
+                        - wait: 30
+                        - assert: {from: 5, to: 4, balance: 6}
```

Comment:
Do you think it will be a good idea to also check the state of the other channels?

Reply:
Thank you for the review Cosmin! In this case I think I would not need to check for the other balances, since I just want to see, if this payment works. The other channels must be updated accordingly, since there is only one path. Would you agree?

Reply:
Thank you for the Review Cosmin! 
In this case I removed all the asserts now. The problem is, I don't even know whether the payment will work or it won't find a path (see expected response code 200 or 409). With this test we could test, whether payments expire or not.